pkg/osvscanner/scan.go

```diff
@@ -110,75 +98,79 @@ func scan(accessors ExternalAccessors, actions ScannerActions) (*imodels.ScanRes
 		plugins = append(plugins, java.NewDefault())
 	}
 
-	// --- Lockfiles ---
-	lockfilePlugins := omitDirExtractors(plugins)
+	scanner := scalibr.New()
 
-	for _, lockfileElem := range actions.LockfilePaths {
-		invs, err := scanners.ScanSingleFileWithMapping(lockfileElem, lockfilePlugins)
-		if err != nil {
+	// Build list of paths for each root
+	// On linux this would return a map with just one entry of /
+	rootMap := map[string][]string{}
+
+	// Also build a map of specific plugin overrides that the user specify
+	// map[path]parseAs
+	overrideMap := map[string]filesystem.Extractor{}
+	// List of specific paths the user passes in so that we can check that they all get processed.
+	//nolint:prealloc // Does not matter in this case
+	var specificPaths []string
```

Comment:
Wouldn't this just be
```go
	specificPaths := make([]string, 0, len(actions.LockfilePaths) + len(actions.SBOMPaths))
```

I feel like we should avoid using `nolint` if possible (or, disable the lint check altogether)